### PR TITLE
feat(event-test): fix event test

### PR DESCRIPTION
**Pull request by biantris**
related https://github.com/biantris/koa-server/issues/8

### CODE REVIEW

File: src/modules/event/__tests__/EventQueries.spec.ts [hunk 1]
```diff
@@ -18,8 +18,19 @@ beforeEach(clearDbAndRestartCounters);
...
-  const event = await createEvent();
-  const anotherEvent = await createEvent();
+  const event = await createEvent({
+    name: "Event",
+    start: "2019-01-01T00:00:00.000Z",
+    end: "2019-01-01T23:59:59.000Z",
+    allDay: true,
+  });
+
+  const anotherEvent = await createEvent({
```
Comment (daniloab) [inline src/modules/event/__tests__/EventQueries.spec.ts:28]: trying names like eventB cold be cool

Comment (daniloab) [reply src/modules/event/__tests__/EventQueries.spec.ts:28]: another also works

**Review by daniloab — APPROVED** [review]

File: src/modules/event/__tests__/EventQueries.spec.ts (hunk 1 above)
Comment (biantris) [reply src/modules/event/__tests__/EventQueries.spec.ts:28]: ok!

File: src/modules/event/__tests__/EventQueries.spec.ts [hunk 2]
```diff
@@ -50,25 +45,19 @@ it("should query all events", async () => {
...
 
   // eslint-disable-next-line
-  console.log("result: ", result.data.events);
+  console.log('result: ', result.data.events.node.name);
 
   expect(result.data.events.edges.length).toBe(2);
 
-  expect(result.data.events.edges[0].node.name).toBe(event.name);
-  expect(result.data.events.edges[1].node.name).toBe(eventB.name);
+  expect(result.data.events.edges.node.name).toBe(event.name);
```
Comment (daniloab) [inline src/modules/event/__tests__/EventQueries.spec.ts]: why edges arent an array?

Comment (biantris) [reply src/modules/event/__tests__/EventQueries.spec.ts]: edges are arrays, thank you!

**Pull request merged**